src/styles/main-tag.scss

```diff
@@ -0,0 +1,7 @@
+.main-tag {
```

Comment:
why main-tag? what does tag mean here?

Reply:
I made paralax effect in footer. It fixed of my viewport. I need to my main container be not transparent. And I need it to have margin bottom like footer height

Reply:
ok, I mean the name
tag is some point to mark the place

Reply:
so you agree to use main-tag name?

Reply:
no, main-container would be more suitable

Reply:
ok